### ISSUE TITLE
Samsung s8 - Backup always returns BIOMETRIC_DISMISSED

### ISSUE THREAD

**Issue by Johnny2528** [NiklasMerz/cordova-plugin-fingerprint-aio#264]
In Samsung Galaxy s8 - Android OS version 9.0, when we use backup in fingerprint-aio, it always returns BIOMETRIC_DISMISSED in error callback function even-though we enter the correct pin.

It should return to success callback function if the entered backup pin is correct.

Attached Cordova info log to refer about the project configuration.

[cordova-info.log](https://github.com/NiklasMerz/cordova-plugin-fingerprint-aio/files/5065849/cordova-info.log)

              function authenticate() {
                     if (typeof window.Fingerprint != 'undefined') {
                         window.Fingerprint.isAvailable(function (res) {
                         window.Fingerprint.show({
                              title: "Biometric plugin",
                              clientId: "Fingerprint-Demo",
                              clientSecret: "clientSecret",
                              fallbackButtonTitle:"use Backup",
                              disableBackup: false,
                              secret: "secret",
                              loadSecret: true
                     }, successCallback, errorCallback);
                         }, function(err){
                              alert("auth Error not available");
                         });
                     }
                 }

                 function successCallback(){
                    alert("Authentication successfull");
                  }
              
                  function errorCallback(err){
                    alert("Authentication invalid " + JSON.stringify(err));
                  }

**Comment by kendyl93** [comment]
Are you sure you are using the right [optionalParameters](https://github.com/NiklasMerz/cordova-plugin-fingerprint-aio#optional-parameters-1)?

**Comment by Johnny2528** [comment]
Yes I tried with multiple combinations (including with and without clientId and clientSecret based on new and old documentations) there is no behaviour change.  

Added to that we have tested it in different devices with face recognition on and off and observed different behaviour for the same app.  Please check the attached document for more details.

[Response in different devices.txt](https://github.com/NiklasMerz/cordova-plugin-fingerprint-aio/files/5099642/Response.in.different.devices.txt)

**Comment by NiklasMerz** [comment]
Sounds like a device specific issue
